components/base/sensorcontrolled/velocities.go

```diff
@@ -215,6 +226,33 @@ func (sb *sensorBase) State(ctx context.Context) ([]float64, error) {
 	return []float64{linvel.Y, angvel.Z}, nil
 }
 
+func (sb *sensorBase) validateControlLoopConfig(ctx context.Context, controlLoopConfig control.Config) error {
```

Comment:
This function doesn't appear to be called from anywhere, and is dead code. Where is it supposed to be used? (Maybe I made a mistake, and just missed where it's already used?)

Reply:
should be called in the constructor just forgot to add it, good catch!